src/core/lib/iomgr/socket_windows.cc

```diff
@@ -39,6 +39,8 @@
 #include "src/core/lib/iomgr/sockaddr_windows.h"
 #include "src/core/lib/iomgr/socket_windows.h"
 
+DWORD grpc_wsa_socket_flags;
```

Comment:
shouldn't there be a function to return the default flags to use?

Reply:
+1 - let's turn this into a function rather than have everyone reference the global directly